cmd/collectors/zapi/plugins/certificate/certificate.go

```diff
@@ -98,18 +100,30 @@ func (my *Certificate) Run(dataMap map[string]*matrix.Matrix) ([]*matrix.Matrix,
 
 		// update certificate instance based on admin vaserver serial
 		for certificateInstanceKey, certificateInstance := range data.GetInstances() {
+			unixTime = time.Now()
 			if certificateInstance.IsExportable() {
-				certificateInstance.SetExportable(false)
+				name := certificateInstance.GetLabel("name")
 				serialNumber := certificateInstance.GetLabel("serial_number")
+				svm := certificateInstance.GetLabel("svm")
+				CertType := certificateInstance.GetLabel("type")
+				certificateInstance.SetLabel("uuid", name+serialNumber+svm)
 
-				if serialNumber == adminVserverSerial {
-					certificateInstance.SetExportable(true)
+				if expiryTimeMetric = data.GetMetric("certificate-info.expiration-date"); expiryTimeMetric == nil {
+					my.Logger.Error().Msg("missing expiry time metric")
+					continue
+				}
+				if expiryTime, ok := expiryTimeMetric.GetValueFloat64(certificateInstance); ok {
+					// convert expiryTime from float64 to int64 and then to unix Time
+					unixTime = time.Unix(int64(expiryTime), 0)
+					certificateInstance.SetLabel("expiry_time", unixTime.UTC().String())
```

Comment:
Use .Format(time.RFC3339) instead of String()

Reply:
Done